functions/ably-jwt/ably-jwt.js

```diff
@@ -0,0 +1,70 @@
+const axios = require("axios");
+const generateAblyJWT = require("./generate-ably-jwt.js");
+
+const jwtSettings = {
+  apiKey: process.env.ABLY_APIKEY,
+  clientId: process.env.ABLY_CLIENTID,
```

Comment:
I think `clientId` needs to be different for each user, perhaps it should be `id` from the URL?

Reply:
Oh great. I was wanting to ask you that.